src/jit/emitxarch.cpp

```diff
@@ -6575,9 +6598,9 @@ void emitter::emitDispIns(
         printf(" %-9s", sstr);
     }
 #ifndef FEATURE_PAL
-    if (strnlen_s(sstr, 10) > 8)
+    if (strnlen_s(sstr, 10) >= 8)
 #else  // FEATURE_PAL
-    if (strnlen(sstr, 10) > 8)
+    if (strnlen(sstr, 10) >= 8)
```

Comment:
I missed the FEATURE_PAL case in the original commit. I wonder if using `strnlen`/`strnlen_s` is warranted here...